### ISSUE TITLE
Use a websocket only lib instead of mongoose

### ISSUE THREAD

**Issue by CardealRusso** [alifcommunity/webui#72]
More than 90% of the compiled WebUI file size is from mongoose, and from that 90% websocket usage should represent less than 5%.

I found a working websocket lib, I was able to compile the exe server with TCC, the lib is also cross-platform. The exe weighed less than 17kb.

https://github.com/bsapundzhiev/ucwebsocket

**Comment by CardealRusso** [comment]
this also closes #61

**Comment by AlbertShown** [comment]
Thank you for your suggestion. WebUI uses Mongoose because it supports the latest WebSockets standards, has a wide choice of lib for encryption using custom APIs, and it has a web-server feature built in, and it's sponsored and used by some important companies. The closest Mongoose WebSocket competitor I found is [libwebsockets.org](https://libwebsockets.org/), but if we used it, we still need another lib for the web server.

You are right, Mongoose code has many other features we don't need in WebUI, but we can easily add some flags to the compilers/libkers to remove unused references. I will add those flags soon. Thank you for pointing this out.
